Sources/Commands/Destination.swift

```diff
@@ -140,14 +145,20 @@ public struct Destination {
   #if os(macOS)
     /// Returns the host's dynamic library extension.
     public static let hostDynamicLibraryExtension = "dylib"
+
+    /// Target subpath for the host system.
+    private static let hostTargetSubpath = "macosx-x86_64"
   #else
     /// Returns the host's dynamic library extension.
     public static let hostDynamicLibraryExtension = "so"
+
+    /// Target subpath for the host system.
+    private static let hostTargetSubpath = "linux-x86_64"
```

Comment:
I'm not sure about inventing a new subpath key here... using the `target` would be more straightforward I think?

The entire triple situation is a mess, so its best to either just directly use the same values, or if not we need to invent something a little more rigorous for our own uses.